src/main/resources/data/computercraft/lua/rom/apis/textutils.lua

```diff
@@ -335,6 +335,42 @@ empty_json_array = mk_tbl("[]", "empty_json_array")
 -- @see textutils.unserialiseJSON
 json_null = mk_tbl("null", "json_null")
 
+local serializeJSONString
+do
+    local function hexify(c)
+        return "\\u00" .. string.format("%02X", string.byte(c))
+    end
+
+    local hpat = string.format("[%c-%c]", 0x7F, 0xFF)
```

Comment:
I'd just replace this with `[\x7f-\xff]`, and inline it into the pattern below.

Reply:
Actually, reading through the spec, it's not clear if we need this or not. Anything in the `\x20` to `\x10ffff` range is valid, which definitely includes all of CC's characters.

Reply:
Yep, it's valid, but it's unclear what encoding we use to send jsons and what encoding receiving side expects. Spec recommends to encode in UTF-8 as the best guess, but we can safely encode everything using only ASCII. Python json encoder escapes everything from 0x7f as well, unless you explicitly disable this.